test/coverageCodeLensProvider.test.ts

```diff
@@ -0,0 +1,92 @@
+import * as vscode from 'vscode';
+import * as fs from 'fs'
+import { describe, expect, it, vi, beforeEach, afterEach } from 'vitest'
```

Comment:
Tests are now executed by https://github.com/microsoft/vscode-test-cli with mocha, so I didn't think this would work. Did you run `pnpm test`?